packages/core/src/components/datetime/stories/datetime.args.ts

```diff
@@ -186,6 +186,22 @@ export const DatetimeStoryArgs = {
         category: Category.SLOTS,
       },
     },
+    dateTarget: {
+      control: 'text',
+      description:
+        'The date value displayed when used with useButton. If not provided, the default date format will be shown.',
+      table: {
+        category: Category.SLOTS,
+      },
+    },
+    timeTarget: {
```

Comment:
@matheus-rosa-jsm Could you add an explicit example for it in Storybook?

Reply:
Added to "Using With Button" section